build/getcli/jf.sh

```diff
@@ -13,11 +12,11 @@ else
     echo "Downloading the latest version of JFrog CLI..."
 fi
 
-if $(echo "${OSTYPE}" | grep -q msys); then
+if echo "${OSTYPE}" | grep -q msys; then
```

Comment:
are you sure?

Reply:
I manually checked in multiple OSs - MacOS, Ubuntu, Debian, Alpine, Fedora.
Also, the automatic tests check for - MacOS, Ubuntu, Windows.